packages/api/src/routers/aggregator/root.ts

```diff
@@ -34,8 +34,8 @@ const bulkInsert = async (
         );
         break;
       default: {
-        const rrwebEvents = [];
-        const logs = [];
+        const rrwebEvents: any[] = [];
+        const logs: any[] = [];
```

Comment:
These two should be ` VectorLog[]` and data below is `VectorLog[]` as well

Reply:
`(VectorLog | VectorMetric)[]`?

Reply:
Yeah, maybe need some better refactoring. the `VectorMetric[]` is in the previous case block. I believe here we want to cast `data as VectorLog[]` instead

Reply:
got it. added casting `data as VectorLog[]` in `default` block, but ideally we want to add zod validation and re-do this fn type signature to something like:

```ts
(
  hdxTelemetry: "log" | undefined,
  apiKey: string,
  data: VectorLog[],
)  |  (
  hdxTelemetry: "metric",
  apiKey: string,
  data: VectorMetric[],
) 
```